FirebaseAuth/Tests/Unit/AuthTests.swift

```diff
@@ -1890,6 +1890,9 @@ class AuthTests: RPCBaseTests {
       waitForExpectations(timeout: 5)
       waitForAuthGlobalWorkQueueDrain()
 
+      // Time for callback to run.
+      RPCBaseTests.waitSleep()
```

Comment:
Is there a limit for this that we can pass in, or is it implicit?

Reply:
It's implicit, intended to a big enough time for ready async functions to fire